src/utils/utils.ts

```diff
@@ -1,65 +1,93 @@
 export const cleanMarketsData = (
-  response: { id: string; marketname: any }[], distance: number
-) => {
+  response: { id: string; marketname: string }[], distance: number ) => {
```

Comment:
For dev empathy/readability, can these types be moved into an interface?

Reply:
yes. That's the plan for a future refactoring branch/PR.